examples/with-universal-redirect/pages/_error.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import ErrorComponent from 'next/error'
+import redirect from 'next-universal-redirect'
```

Comment:
Shall we make this open? I can't see the source.
See: https://github.com/timneutkens/next-redirect